src/commands/init/cold_storage.md

```diff
@@ -7,17 +7,25 @@ Glacier or OVH Cloud Archive.
 
 To use a cold storage and not access any data in the storage for every-day
 operations, rustic needs an extra repository to store hot data. This repository
-can be specified by the `--hot-repo` option or the `RUSTIC_REPO_HOT`
-environmental variable, e.g.:
+can be specified by the `hot-repo` option or the `RUSTIC_REPO_HOT` environmental
+variable, e.g.:
+
+```toml
+[repository]
+repository = "rclone:foo:cold-repo"
+repo-hot = "rclone:foo:hot-repo"
```

Comment:
rclone vs. opendal, do we want to recommend external services or rather focus on supporting embedded one's?

Reply:
I think it should be at user's choice, but the opendal example I've added also shows how to set individual options for hot/old repo parts.